packages/grid_client/tests/modules/applications/casperlabs.test.ts

```diff
@@ -46,10 +44,10 @@ test("TC2683 - Applications: Deploy Casperlabs", async () => {
   //Test Data
   const name = "gw" + generateString(10).toLowerCase();
   const tlsPassthrough = false;
-  let cpu = generateInt(1, 4);
-  let memory = generateInt(1, 6);
-  let rootfsSize = generateInt(0, 2);
-  let diskSize = generateInt(10, 50);
+  const cpu = 2;
+  const memory = 4;
+  const rootfsSize = 0;
```

Comment:
This shouldn't be 0. Can we set it to 2 instead?

Reply:
This might not work all the time. Rootfs is computed on the CPU & memory size. So, if the CPU & memory > 0, rootfs will be more than 0. The VM might fail as we didn't provide the correct capacity it needs while filtering nodes. (The requested capacity is less than the needed capacity)

Reply:
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2784/commits/d3bdda7299fbc574bd67833d5b24164aba18f714